### PR TITLE
Updated tutorials

### PR DESCRIPTION
**Pull request by MMenchero**
## Description: 

Updated versions of the following tutorials: 
- Getting Started 
- FAQ
- Historical Forecast 
- Irregular Timestamps 
- Bitcoin Prediction (Example)

These updates are a part of a larger initiative to improve TimeGPT’s documentation.

### CODE REVIEW

File: nbs/docs/misc/0_faqs.ipynb [hunk 1]
```diff
@@ -35,29 +35,62 @@
    "cell_type": "markdown",
```
Comment (mergenthaler) [inline nbs/docs/misc/0_faqs.ipynb:35]: <p>Great! But I all the accordeons have <code>Details</code> as title or summary and that is very confusing. You can try using the <code>summary </code>  tag:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">&lt;details&gt;
  &lt;summary&gt;&lt;i&gt;Wow, so fancy&lt;/i&gt;&lt;/summary&gt;
  &lt;b&gt;WOW, SO BOLD&lt;/b&gt;
&lt;/details&gt;
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/nixtla/pull/316/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Comment by MMenchero** [comment]
Added Bitcoin Price Prediction example.

**Comment by MMenchero** [comment]
Converting to draft until all the checks have passed.

**Comment by MMenchero** [comment]
This is ready for review @AzulGarza @mergenthaler.

File: nbs/docs/misc/0_faqs.ipynb (hunk 1 above)
Comment (MMenchero) [reply nbs/docs/misc/0_faqs.ipynb:35]: The text already had this structure. Below is how it appears in the preview. This issue should be resolved once the notebook is rendered in the documentation.
Screenshot 2024-04-29 at 16 59 28

**Review by mergenthaler — APPROVED** [review]
Thanks!

**Comment by MMenchero** [comment]
<p>I think we should keep this here. We are incorporating exogenous variables, but the discrepancy between quarterly revenue data and daily stock price data results in irregular timestamps.</p>

---

 <a href='https://app.reviewnb.com/Nixtla/nixtla/pull/316/discussion/#comment-2078548272'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Pull request merged**